README.md

```diff
@@ -1,59 +1,41 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome books
 
-> One paragraph statement about the project.
+> A webpage to display books
 
 ![screenshot](./app_screenshot.png)
```

Comment:
- [OPTIONAL] The link to the screenshot is broken, please kindly fix that as this will make your readme look more professional.